CONTRIBUTING.md

```diff
@@ -123,7 +123,7 @@ npm install or yarn install
 #### Package List
 
 **1. Client**
-This is a  wrapper written on top of the ```request``` module to suite the sendgrid module. All requests made to the sendgrid API are invoked by the `request` function in the `client.js`.
+This is a  wrapper written on top of the ```request``` module to suite the Sendgrid module. All requests made to the Sendgrid API are invoked by the `request` function in the `client.js`.
```

Comment:
Please use Pascal Case for `SendGrid`

Reply:
Sorry, looks like I was a commit behind! Have pushed the capitalisation fix and, I think, an apostrophe correction.